apps/schematic/api/docker-compose.yml

```diff
@@ -1,8 +1,11 @@
-version: "3.8"
+version: "3.10"
 
 services:
   schematic-api:
-    image: ghcr.io/sage-bionetworks/schematic-api:local
+    # image: ghcr.io/sage-bionetworks/schematic-api:local
+    build: 
+      dockerfile: Dockerfile
+    image: sagebionetworks/schematic-aws-api:latest
```

Comment:
This breaks a few standards that the monorepo attempts to establish:
- images should ideally be pushed to GHCR
- the name of the image should ideally be the name of the project in the monorepo
- the project should define "tasks" in its project.json to build, scan and publish the image, hence the image build should not be defined in the docker-compose file.

Reply:
Thanks for the feedbacks. But I wonder what are the reasons that we don't build docker files locally using docker compose..

Reply:
Thanks for responding to this question on slack. I will keep this part as pulling docker image from GHCR then.

Reply:
I have reverted changes in docker-compose.yml other than updating the port number. Mark this as "resolved" for now.